web-app/src/app/screens/Feed/index.tsx

```diff
@@ -332,7 +340,10 @@ export default function Feed(): React.ReactElement {
                     />
 
                     {feed?.data_type === 'gtfs_rt' && (
-                      <AssociatedFeeds feeds={relatedFeeds} />
+                      <AssociatedFeeds
+                        feeds={relatedFeeds}
+                        gtfsRtFeeds={relatedGtfsRtFeeds}
```

Comment:
We should filter out the _current_ feed ID. Otherwise, the list of associated feeds will also contain the feed itself.
<img width="1514" alt="Screenshot 2024-07-23 at 1 35 42 PM" src="https://github.com/user-attachments/assets/293ed68e-a2b8-4b6b-af32-ce8be36b550b">

Reply:
Made this change